guide/12-enrich-data-with-thematic-information/part4_what_to_enrich_datacollections_analysisvariables.ipynb

```diff
@@ -1 +1 @@
ABaf4/2Hxu+G+h+GvECW/hu60J7fW7uexnRrm5jnLRTxWV6BCwjCzRlMvwQRs2v8hRfs8SeHvib8Lv2dfD+sWl9JFoFrB4+t7SNkQ6XoN0moaUFUM3kQm8kls4YJTmazmmALCBqbxNRdWCwVFr4V9yPtH4q+NR4O+J/hr4SeAfAun+INW8RaZqurA3E8VhDDb6TNYQSZf7POWd31CPaAoGFbJ6V3dpr3wr0280Xwp8RY/D+geKdbjL2+jPc2zzysv3xbh1iknC92WP8AAV4J+1V+zhr/AMYfi14I+Ilp4J8H/EHTfDela7p9xpPjC4e3hWfVJtLlguoMaZqiF41sZY2zGhAl+ViCwrnNY/Zz+Lk3xG8H+NfAOg+G/BosbbQ7HVotP1Ka4s49O0i+luPsEWnyaXHb3CRxTSizuE+xTW8shb5kRUK+sVL7sFg6DS91fd/X5H0raeMfgtrniiz8K+DhoWtTyXBgvBaXFi8loTby3EZkhDeYxkERCqqlsZfGxWI+y/hhpmm6VoE1vpdvHbRtcMxWJFQElEGcKAM4Ar82/hp+zNrvgDRvhfpdudOh/wCEM1fVtT1IW5cLM2pW9/GWi/dLvkaW7V5C+z+I5JwD+mPw/wD+QNL/ANdm/wDQVrmxtWcqdpM9DKqFONe8Etj/2Q==\"/>"],"text/plain":["<IPython.core.display.Image object>"]},"execution_count":67,"metadata":{},"output_type":"execute_result"}],"source":["covid_map = gis.map('USA', 4)\n","covid_map"]},{"cell_type":"code","execution_count":61,"metadata":{},"outputs":[{"data":{"text/plain":["True"]},"execution_count":61,"metadata":{},"output_type":"execute_result"}],"source":["# Plot on a map\n","covid_map.remove_layers()\n","new_df.spatial.plot(map_widget=covid_map,\n","                    renderer_type='c',  # for class breaks renderer,\n","                    method='esriClassifyNaturalBreaks',  # classification algorithm,\n","                    class_count=5,  # choose the number of classes,\n","                    col='FEM65',  # numeric column to classify,\n","                    cmap='viridis',  # color map to pick colors from for each class,\n","                    alpha=0.7)"]},{"cell_type":"markdown","metadata":{},"source":["## Conclusion"]},{"cell_type":"markdown","metadata":{},"source":["In this part of the `arcgis.geoenrichment` module guide series, you saw how `data_collections` property of a `Country` object lists its available `data_collection`s and `analysis_variable`s. You explored different data collections, their analysis variables and then enriched study areas using the same. Towards the end, you experienced how spatially enabled dataframes can be enriched.\n","\n","In the subsequent pages, you will learn about Generating Reports and Standard Geography Queries."]}],"metadata":{"anaconda-cloud":{},"kernelspec":{"display_name":"arcgispro-dev","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.9.11 [MSC v.1931 64 bit (AMD64)]"},"livereveal":{"scroll":true},"toc":{"base_numbering":1,"nav_menu":{},"number_sections":true,"sideBar":true,"skip_h1_title":true,"title_cell":"Table of Contents","title_sidebar":"Contents","toc_cell":true,"toc_position":{"height":"calc(100% - 180px)","left":"10px","top":"150px","width":"274px"},"toc_section_display":true,"toc_window_display":true},"vscode":{"interpreter":{"hash":"07c13af76457a6f4e5d5b34d0e1bc42b2e017343b05b916c8871f483eed35ce6"}}},"nbformat":4,"nbformat_minor":2}
```

Comment:
<h3>Data Collections for the U.S. </h3><p> </p><p>The <code>data_collections</code> property of a <code>Country</code> object lists its available data collections and analysis variables under each data collection as a Pandas dataframe. </p><p>In order to discover the data collections for a particular country, first access the reference variable to it using the <code>country.get()</code> method, then fetch the data collections from <code>country.data_collections</code> property. Once we know the data collection we would like to use, we can look at the <code>analysisVariables</code> available in that data collection.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>